src/observer.rs

```diff
@@ -252,8 +252,11 @@ impl ObserverData {
     /// let cie1931_d50_xyz = colorimetry::observer::CIE1931.xyz_d50();
     /// approx::assert_ulps_eq!(cie1931_d50_xyz.values().as_ref(), [96.421, 100.0, 82.519].as_ref(), epsilon = 5E-2);
     ///
+    /// # #[cfg(feature = "supplemental-observers")]
+    /// # {
```

Comment:
This is completely unrelated to the `unused_variables` lint. But I found that this did not compile when running the ignored tests with some combinations of features.